pyiron_contrib/protocol/jobs/decoupled_oscillators.py

```diff
@@ -124,42 +160,35 @@ def _set_base_structure(self):
         self._base_atom_ids = np.delete(np.arange(len(self.input.structure)).astype(int),
                                         self.input.oscillators_id_list)
 
-    def _create_base_job(self, pr=None, name=None):
+    @property
+    def _base_name(self):
+        return self.job_name + '__base'
```

Comment:
I'd rather just want to leave the extension as '_base' than '__base', unless there's a specific reason for the dunder?

Reply:
I always dunder auto generated names. It's not a guarantee but adds a level of safety. Ie imagine a project where Someone creatively names this job "job" and then later in the project in an equally impressive bout of creativity calls something else "job_base". So my rule of thumb is to dunder auto names